src/job/job.spec.js

```diff
@@ -167,4 +167,28 @@ describe("job", () => {
       ).rejects.toMatchSnapshot();
     });
   });
+
+  describe(".runSequence() :", () => {
+    let id;
+
+    afterEach(async () => {
+      await xo
+        .call("vm.delete", { id, delete_disks: true })
+        .catch(error => console.error(error));
+    });
+
+    it("runs a job", async () => {
+      const jobId = await xo.createTempJob(defaultJob);
+      const snapshots = xo.objects.all[config.vmIdXoTest].snapshots;
+      await xo.call("job.runSequence", { idSequence: [jobId] });
+      await waitObjectState(
+        xo,
+        config.vmIdXoTest,
+        ({ snapshots: actualSnapshots }) => {
+          expect(actualSnapshots.length).toBe(snapshots.length + 1);
```

Comment:
Instead of testing the size, we can simple test the id bellow.

Reply:
Do you mean I have to do : 

```
await waitObjectState(
    xo,
    config.vmIdXoTest,
    ({ snapshots: actualSnapshots }) => {
      const diffSnapshots = difference(actualSnapshots, snapshots);
      expect(diffSnapshots.length).toEqual(1);
      id = diffSnapshots[0];
   }
);
```

Reply:
`id === undefined` means that the snapshot was not be created.

Reply:
But I think `id` cannot be `undefined` if I test the size of `diffSnapshots` before.

Reply:
We'll  got anyway the id. Why not simply test it instead of comparing the arrays size?

Reply:
But comparing the size before is better to not do a useless  loop. I'll resolve this conversation.

Reply:
because I think we have to test the size of `diffSnapshots` in case of there are 2 or 3 more snapshots instead of 1...

Reply:
If it will have an other snapshots. The diff will not be correct. Well be not sure to delete the snapshot created by this test. And this test will not be consistent.

Reply:
But it will not be the case because this test will just add one snapshot.

Reply:
So what do I have to do ? no tests with the size ?
In case of having more than 1 snapshot, I can put an array of snapshots to delete in the `afterEach`.